### ISSUE TITLE
Script pic scaled wrong in retina mode

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap--Build-Your-Own-Blocks#1378]
On my shiny new retina Pro,I tried making a script pic and it came out more than double the correct size.  Turning off the retina checkbox fixes it.

This isn't exactly urgent b/c there's an easy workaround, but it's not exactly low priority either b/c it's a pain for us curriculum writers.

**Comment by cycomachead** [comment]
BH got a new computer. 😮

**Comment by cycomachead** [comment]
I'm not sure if there's a great work around for this... :/ 

The problem is Snap<em>!</em> _is_ exporting at the real size...which is exactly 4x the pixels.

**Comment by jmoenig** [comment]
there actually *is* an easy way to "fix" (quote - unquote) this, and I've already tried it, it's simply calling ```normalizeCanvas()``` on the pic before it gets exported. OTOH, *some* people like to export high-res pics and scale them down in their own documents. This is a hard decision. For now I advoate leaving it as it is, i.e. exporting the real resolution.

P.S. hi from Tarragona (Spain)

**Comment by cycomachead** [comment]
Right! I forgot we had that in there!

But yes, this was my feeling too. The only reason I side with Brian on this one is that it _looks_ like blocks are much bigger, since the retina canvas isn't being auto-scaled up, but the regular images of blocks still are, and they would be in other places used on the web too.

**Comment by bromagosa** [comment]
Hey Jens, completely off-topic, but if you pass by BCN let me know!

**Comment by brianharvey** [comment]
When I import a costume in retina mode, it doesn't come out too small.  So exporting shouldn't be too big!

But I'd settle for a checkbox that appears if retina mode is on.

**Comment by cycomachead** [comment]
> When I import a costume in retina mode, it doesn't come out too small. So exporting shouldn't be too big!

Yeah, the more I think about this -- Brian is right here. While large things are nice, you can still get that (at least for blocks) by zooming as you would currently expect to do anyway.

Well, costumes should just be exported at the size they were imported at. (Which I guess means shrinking them down.)
